core/src/main/java/org/commonjava/indy/core/ctl/NfcController.java

```diff
@@ -119,33 +109,22 @@ public NotFoundCacheDTO getMissing( final StoreKey key )
         return doGetMissing( key );
     }
 
-    public Pagination<NotFoundCacheDTO> getMissing( final StoreKey key, int pageIndex, int pageSize )
+    public Pagination<NotFoundCacheDTO> getMissing( final StoreKey key, GenericPagination pagination )
```

Comment:
I get you want to wrap index/pagesize in the second GenericPaination. This is good idea. But it confuses with the normal Pagination<T> interface. How about naming it simply as Page. And I don't think it needs to be extendable. So NfcPagination is not necessary, IMO.

Reply:
@ruhan1 Yep, that's good suggestion, NfcPagination is a little excessive here, at least for current Page is enough. thanks for your advice :) I'll update in the following commit.